<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-04T10:06:31 IST" name="Default suite" finished-at="2024-04-04T10:06:37 IST" duration-ms="5422">
    <groups>
    </groups>
    <test started-at="2024-04-04T10:06:31 IST" name="Default test" finished-at="2024-04-04T10:06:37 IST" duration-ms="5422">
      <class name="extent.TestScript">
        <test-method is-config="true" signature="configReports()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:31 IST" name="configReports" finished-at="2024-04-04T10:06:31 IST" duration-ms="156" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configReports -->
        <test-method is-config="true" signature="launchBrowser()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:31 IST" name="launchBrowser" finished-at="2024-04-04T10:06:36 IST" duration-ms="4186" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="login(java.lang.reflect.Method)[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:36 IST" name="login" finished-at="2024-04-04T10:06:36 IST" duration-ms="10" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void extent.TestScript.clickOnBooks()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="clickOnBooks()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:36 IST" name="clickOnBooks" finished-at="2024-04-04T10:06:36 IST" duration-ms="846" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User failed to click on BOOKS link expected [Demo Web Shop] but found [Demo Web Shop. Books]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User failed to click on BOOKS link expected [Demo Web Shop] but found [Demo Web Shop. Books]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at extent.TestScript.clickOnBooks(TestScript.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnBooks -->
        <test-method is-config="true" signature="logout()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:37 IST" name="logout" finished-at="2024-04-04T10:06:37 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:37 IST" name="closeBrowser" finished-at="2024-04-04T10:06:37 IST" duration-ms="229" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="reportBackup()[pri:0, instance:extent.TestScript@71d44a3]" started-at="2024-04-04T10:06:37 IST" name="reportBackup" finished-at="2024-04-04T10:06:37 IST" duration-ms="309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportBackup -->
      </class> <!-- extent.TestScript -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
